pkg/cliplugins/bind/plugin/bind_compute.go

```diff
@@ -176,7 +176,8 @@ func (o *BindComputeOptions) Run(ctx context.Context) error {
 	}
 
 	// wait for bind to be ready
-	if !bindReady(bindings, placement) {
+	message := ""
+	if ready, _ := bindReady(bindings, placement); !ready {
```

Comment:
I think you could do `if ready, message := bindReady(bindings, placement)`

Reply:
Fixed